### PR TITLE
Update rubocop

### PR DESCRIPTION
**Pull request by szb512**
### Summary

Bumps rubocop to 0.68.1.

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -25,6 +25,6 @@ checks:
 plugins:
   rubocop:
     enabled: true
-    channel: rubocop-0-67
+    channel: rubocop-0-68
```
Comment (composerinteralia) [inline .codeclimate.yml]: From what I can tell codeclimate/codeclimate-rubocop doesn't have this channel yet, so this doesn't seem like it would work (I'm looking at https://github.com/codeclimate/codeclimate-rubocop/branches/all?utf8=%E2%9C%93&query=channel%2Frubocop).

We should also `bundle update rubocop` so the version in the Gemfile.lock matches this.

File: .whitesource [hunk 2]
```diff
@@ -0,0 +1,8 @@
+##########################################################
```
Comment (composerinteralia) [inline .whitesource]: This whitesource config seems unrelated to Rubocop. I don't think that belongs in this PR.

File: .codeclimate.yml (hunk 1 above)
Comment (szb512) [reply .codeclimate.yml]: I clearly know this version exists on github.

File: .codeclimate.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-version: "2"
+version: "1"
```
[the remark this answers is not in the document]
Comment (szb512) [reply .codeclimate.yml]: A newer version is 0.68.1 with 1 being the last version number.

File: .whitesource (hunk 2 above)
Comment (szb512) [reply .whitesource]: True. I can't remove it. If I could, it would be recreated.

File: .codeclimate.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (composerinteralia) [reply .codeclimate.yml]: Looking at https://docs.codeclimate.com/docs/advanced-configuration, this seems to be the codeclimate version number, not the rubocop patch version number.

File: .codeclimate.yml (hunk 1 above)
Comment (composerinteralia) [reply .codeclimate.yml]: This is not clear to me. Can you offer some support that it is true? The link I provided doesn't show that codeclimate has a rubocop-0-68 channel yet.

Comment (composerinteralia) [reply .codeclimate.yml]: Also, this is what codeclimate is showing:

Screen Shot 2019-05-01 at 8 27 20 AM

Comment (koic) [reply .codeclimate.yml]: Yes. CodeClimate does not yet support RuboCop 0.68.
https://github.com/codeclimate/codeclimate/blob/e31c097b4069b84106c4c074be62e05fce96d801/config/engines.yml#L177-L201

**Comment by kamipo** [comment]
Closing since CodeClimate does not support RuboCop 0.68 yet.
Please update the CodeClimate side first.
